tools/c7n_index/c7n_index/metrics.py

```diff
@@ -87,14 +87,18 @@
                     }
                 }
             ]
-            
-            
+
+
         },
         'accounts': {
             'type': 'array',
             'items': {
                 'type': 'object',
-                'required': ['name', 'bucket', 'regions', 'title'],
+                'anyOf': [
```

Comment:
anyOf: **at least one (or more)** should match 
oneOf: **only one** should match 

They are both valid and it boils down to personal choice.